ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-rosa.yaml

```diff
@@ -33,16 +33,22 @@ tests:
+          {"step": "rosa-cluster-provision", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-cluster-wait*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-sts-account-roles-create", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-cluster-deprovision", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-sts-account-roles-delete", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},          
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "cucushift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "camel-k-interop-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "CMLK", "group": {"name": "lp-tests", "priority": 3}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]}
         ]
       FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
       FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OCM_LOGIN_ENV: staging
```

Comment:
This value is set to `staging` by default as a part of the workflow, but I've seen our team still mention it in other scenarios, so keeping it for consistency

& Also default values can be changed by maintainers, so mention it in the config file helps

Reply:
I completely agree I think placing it here helps it be easier to understand without someone needing to go digging to the step defaults +1